front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/tag/DefaultEntityTagsDAO.java

```diff
@@ -51,17 +52,6 @@ private EntityTags upsert(String id, EntityTags tag) {
   }
 
   @Override
-  public Collection<EntityTags> all() {
```

Comment:
I'm somewhat on the fence as to whether it's worth the overhead to maintain an in-memory cache of all tags ... this may not scale in the long-term but does provide a reasonably efficient answer to the "I need to re-index everything that's in S3".

The downside is that we're maintaining and refreshing this cache with no real caller beyond the once in a blue moon reindex operation.

Alternative could be that we treat the re-index as a special case that we run locally.